src/OpenTelemetry.PersistentStorage.FileSystem/README.md

```diff
@@ -14,40 +14,52 @@ exporters to improve the reliability of data delivery.
 dotnet add package OpenTelemetry.PersistentStorage.FileSystem
 ```
 
-## Basic Usage
+## Usage
 
 ### Setup FileBlobProvider
 
 ```csharp
-using var persistentBlobProvider = new FileBlobProvider("test");
+using var persistentBlobProvider = new FileBlobProvider("@C:\temp");
 ```
 
+#### Configuration
+
 Following is the complete list of configurable options that can be used to set
 up FileBlobProvider:
 
-* `path`: Sets folder location where blobs are stored.
+##### path
+
+Sets folder location where blobs are stored.
```

Comment:
```suggestion
Sets folder location where blobs are stored. Required.
```

I'm not going to drop comments on all the other things, but maybe add "Optional" to them? Like...

```
##### maxSizeInBytes

Maximum allowed folder size. Optional. Default if not specified: `50 MB`.
```

Reply:
updated